error.go

```diff
@@ -0,0 +1,19 @@
+package etcd
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return microerror.Cause(err) == invalidConfigError
+}
+
+var multipleValuesError = microerror.New("multiple values")
```

Comment:
There is an argument that this should be in `microstorage`, but I don't see the need right now.